deploy/crds/org_v1_che_cr.yaml

```diff
@@ -33,6 +33,8 @@ spec:
     # when set to true the operator will attempt to get a secret in OpenShift router namespace
     # to add it to Java trust store of Che server. Requires cluster-admin privileges for operator service account
     selfSignedCert: false
+    ## If enabled then the certificate from `custom-public-cert` config map will be added to Java trust store of Che server.
+    customPublicCert: false
```

Comment:
I know you did in the same way as we have for selfSignedCert and gitSelfSignedCert but it would be much more flexible if user configures configMap/secretMap name instead of specifying `true`/`false` and store their certs in hard-coded `custom-public-cert` config map. WDYT?

Reply:
done